test/dataBuilders.js

```diff
@@ -1,5 +1,13 @@
 import { build, fake } from '@jackfranklin/test-data-bot';
 
+const ramdomWords = (wordsNumber) => {
```

Comment:
This function can be simplified, take this as an example:

```js
Array.from({ length: 3 }, () => Math.random())
```

Reply:
I didn't know that existed, thats cool! 
I refactored a little so it uses the faker.js API `Array.from({ length: 3 }, () => fake((f) => f.lorem.word()))`